src/moonstone-samples/lib/DataGridListSample.js

```diff
@@ -110,12 +151,70 @@ module.exports = kind({
 		this.modelIndex = idx;
 		return records;
 	},
+	generateDataGridList: function (itemType) {
+		if (this.$.gridList) {
+			this.$.gridList.destroy();
+		}
+
+		var props = utils.clone(this.dataListDefaults),
```

Comment:
Simplest solution here is to drop `dataListDefaults` and assign the object literal to props. There's really no need for it to be a class member and is only used in this method.